src/util/brush/getCircle.js

```diff
@@ -8,7 +8,7 @@
  * @param  {number} columns    The number of columns.
  * @param  {number} [xCoord = 0] The x-location of the center of the circle.
  * @param  {number} [yCoord = 0] The y-location of the center of the circle.
- * @returns {number[][]}        Array of pixels contained within the circle.
+ * @returns {Array.number[]}        Array of pixels contained within the circle.
```

Comment:
🤔 This is probably correct, but it's not what I would have guessed

Reply:
The `.` syntax is soon to be (or already is?) deprecated in  `jsdoc3`as far as I know, its `google/closure-compiler` syntax. You can see arrays defined as `string[]` [here](http://usejsdoc.org/tags-param.html), and here is a note that `number[][]` is standards compliant: https://github.com/jsdoc3/jsdoc/issues/705

Reply:
Here is another more recent example of Hegemonic ignoring close-compiler syntax (by ignoring `dandv`'s suggestion and saying there is no way to document individual array elements in jsdoc3 https://github.com/jsdoc3/jsdoc/issues/1073

I'm happy to discuss any of this and don't mean to strong-arm. The docs for params are pretty bare, but `type[]` and `type[][]` do seem to be the standard.

Reply:
I don't have a great deal of experience with JSDoc so happy for whatever you say is the right thing to do here. The linter was complaining that `number[][]` was a "JSDoc syntax error". Hence the change.

Reply:
Aha, that's interesting. The two aren't necessarily in sync with each other. If eslint prefers your syntax, and the API docs still autogenerate fine... maybe we should use that instead. Like I said the docs aren't super clear on this. I'm not opinionated either way, lets wait for @dannyrb  to chime in.

Reply:
@mikehazell, I'm okay with keeping your changes here "as is" for now. We'll want to create a follow-up research item on this one. 

If `Docma` is happy with the new syntax, but the linter is not, we may need a custom lint rule/modification to acoomodate. If `Docma` is also unhappy, then I see no immediate need to change, but I'd want to create a related issue in `Docma`'s repo.